### PR TITLE
Create "Tech Maps" Stub

### PR DESCRIPTION
**Pull request by paper-toaster**
Creates a stub for the term "Tech map" ala #3143 and #3151.

**This stub covers the use of a hotly debated term; if you have an opposing view to the description listed in this article, please leave a comment and explain your reasoning**

### To-Do:
- [x] Create and add images
- [x] Replace placeholders
- [x] Reach a final verdict on how to handle definition of "Tech map"
- [x] Write said definition
- [x] Check for typos and such

*Credits to @Nivalyx and K4L1 for helping out with the description*

*K4L1 gathered all the images (and then some) for me as well, so kudos to you man*

*Just realized that this pr attracted the attention of a well-known mapper lol*

### CODE REVIEW

**Comment by Oedipsos** [comment]
Hey @paper-toaster , thanks for your work on this stub.

I just noticed a typo at line 47 where you wrote `all` twice.
Futhermore, I would delete the second `more` at line 45 to be consistent with the text at line 47.
Besides that, this stub seems nice to me.

For the `History` section, it would be nice to write one, but I don't really see what may fit in that section to be honest.

File: wiki/Beatmaps/Technical_maps/en.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+- Irregular slider shapes
+- Quick and sharp changes in slider speed
+- Overall high intensity
+  - Extremely dense in object count throughout large parts of the map (excluding [streams](/wiki/Beatmaps/Pattern/Stream))
+
+![Non-generic beatmap pattern (Nhelv)](img/nongeneric_beatmap_pattern.jpg "Non-generic beatmap pattern")
+
+![Hard flow (Densetsu no Matsuri)](img/beatmap_hard_flow.jpg "Beatmap pattern with a hard flow")
+
+Good examples of tech maps that fit this definition are [Silentroom - Nhelv](https://osu.ppy.sh/beatmapsets/917915#osu/2009432) and [RUMI - Densetsu no Matsuri](https://osu.ppy.sh/beatmapsets/781683#osu/1641637).
```
Comment (MegaApplePi) [inline wiki/Beatmaps/Technical_maps/en.md:44]:
```suggestion
Good examples of tech maps that fit this definition are [Silentroom - Nhelv (Nyxa) \[iniquitatem\]](https://osu.ppy.sh/beatmapsets/917915#osu/2009432) and [RUMI - Densetsu no Matsuri (Net0) \[Oni\]](https://osu.ppy.sh/beatmapsets/781683#osu/1641637).
```

File: wiki/Beatmaps/Technical_maps/en.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+- Irregular slider shapes
+- Quick and sharp changes in slider speed
+- SV sliders
+- Difficult-to-read beatmap patterns (often described as having a hard "flow") (example shown below)
+
+![Difficult-to-read-pattern (Camelia)](img/difficult_to_read_beatmap.jpg "Difficult-to-read pattern")
+
+![Irregular slider shape (MARENOL)](img/irregular_slider.jpg "Irregular slider shape (SV slider)")
+
+Good examples of tech maps that fit this definition are [Camelia - Exit This Earth's Atomosphere (Camellia's "PLANETARY//200STEP" Remix)](https://osu.ppy.sh/beatmapsets/855677#osu/1787848) and [LeaF - MARENOL](https://osu.ppy.sh/beatmapsets/1136149#osu/2404722).=
```
Comment (MegaApplePi) [inline wiki/Beatmaps/Technical_maps/en.md:59]:
```suggestion
Good examples of tech maps that fit this definition are [Camelia - Exit This Earth's Atomosphere (Camellia's "PLANETARY//200STEP" Remix) (ProfessionalBox) \[Primordial Nucleosynthesis\]](https://osu.ppy.sh/beatmapsets/855677#osu/1787848) and [LeaF - MARENOL (Yugu) \[Extra\]](https://osu.ppy.sh/beatmapsets/1136149#osu/2404722).
```

File: wiki/Beatmaps/Technical_maps/en.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+
+Tech maps are most often associated with song genres that offer complex and fast-paced sounds (e.g., drum-and-bass, dubstep, and drumstep) as they often allow for the mapper to explore each and every individual sound in-detail to the beatmap's advantage. Although music of this type is not explicitly required.
+
+Regardless of definition, many players claim that tech maps are unfair (in terms of skill) due to the main increase in difficulty deriving mostly or solely from the confusing patterns or irregular sliders. Likewise, this means that they are not very popular for gaining large amounts of [pp](/wiki/Performance_Points) at once due to the way the current system works and how it undervalues beatmaps of this nature.
+
+## Definitions
+
+Due to the broad nature of the term "tech map," there are many different aspects that could describe the term. The lists below describe different definitions from popular "types" of players.
```
[the remark this answers is not in the document]
Comment (paper-toaster) [reply wiki/Beatmaps/Technical_maps/en.md:25]: The rest of the article uses American English punctuation, which places commas inside quotation marks

**Pull request merged**